downloads/swan-lake-release-notes/swan-lake-alpha4/swan-lake-alpha4-release-note.md

```diff
@@ -333,17 +354,9 @@ To view bug fixes, see the GitHub milestone for Swan Lake <VERSION> of the repos
 
 ### Developer Tools Updates
 
-#### Language Server 
-
-To view bug fixes, see the [GitHub milestone for Swan Lake <VERSION>](https://github.com/ballerina-platform/ballerina-lang/issues?q=is%3Aissue+is%3Aclosed+milestone%3A%22Ballerina+Swan+Lake+-+Alpha4%22+label%3AType%2FBug+label%3ATeam%2FLanguageServer).
-
-#### New Features
-
-#### Improvements
-
 #### Bug Fixes
 
-To view bug fixes, see the GitHub milestone for Swan Lake <VERSION> of the repositories below.
+To view bug fixes, see the GitHub milestone for Swan Lake Alpha4 of the repositories below.
 
 - [Language](https://github.com/ballerina-platform/ballerina-lang/issues?q=is%3Aissue+is%3Aclosed+milestone%3A%22Ballerina+Swan+Lake+-+Alpha4%22+label%3AType%2FBug+label%3ATeam%2FDevTools)
 - [Update Tool](https://github.com/ballerina-platform/ballerina-update-tool/issues?q=is%3Aissue+is%3Aclosed+label%3AType%2FBug+project%3Aballerina-platform%2F32)
```

Comment:
No bugs are listed in update tool or open API links given here? Should we remove them @keizer619 @lnash94

Reply:
We can remove update tool and @lnash94 will update open api link

Reply:
Shall we replace the OpenAPI link with this link  [https://github.com/ballerina-platform/ballerina-openapi/issues?q=is%3Aissue+is%3Aclosed+milestone%3A%22Ballerina+Swan+Lake+-+Alpha4%22](https://github.com/ballerina-platform/ballerina-openapi/issues?q=is%3Aissue+is%3Aclosed+milestone%3A%22Ballerina+Swan+Lake+-+Alpha4%22)

Reply:
Removed the Update Tool link and updated the OpenAPI link.